CHANGELOG.md

```diff
@@ -4,6 +4,105 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Calendar Versioning](https://calver.org/) starting v20.03.
 
+## [20.07.2] - Unreleased
+[20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.1...v20.03.2
```

Comment:
change to: [20.07.2]: https://github.com/dgraph-io/dgraph/compare/v20.07.2...v20.07.1